azure/test/request.test.ts

```diff
@@ -52,85 +52,66 @@ suite('request', () => {
+        const response = await sendTestRequest({ statusCode: 200, body: '{ "data": "Hello World!" }', contentType: 'text/plain' });
         assert.strictEqual(response.parsedBody, undefined);
     });
 
     test('200, json body, json content type', async () => {
-        const response = await sendTestRequest({ status: 200, body: '{ "data": "Hello World!" }', contentType: 'application/json' });
+        const response = await sendTestRequest({ statusCode: 200, body: '{ "data": "Hello World!" }', contentType: 'application/json' });
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         assert.strictEqual(response.parsedBody.data, 'Hello World!');
     });
 
     test('200, json body, no content type, with bom', async () => {
-        const response = await sendTestRequest({ status: 200, body: `\ufeff{ "data": "Hello World!" }` });
+        const response = await sendTestRequest({ statusCode: 200, body: `\ufeff{ "data": "Hello World!" }` });
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         assert.strictEqual(response.parsedBody.data, 'Hello World!');
     });
 
     test('200, json body, json content type, with bom', async () => {
-        const response = await sendTestRequest({ status: 200, body: `\ufeff{ "data": "Hello World!" }`, contentType: 'application/json' });
+        const response = await sendTestRequest({ statusCode: 200, body: `\ufeff{ "data": "Hello World!" }`, contentType: 'application/json' });
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         assert.strictEqual(response.parsedBody.data, 'Hello World!');
     });
 
     test('400', async () => {
-        await assertThrowsAsync(async () => await sendTestRequest({ status: 400 }), /400/);
+        await assertThrowsAsync(async () => await sendTestRequest({ statusCode: 400 }), /400/);
     });
 
     test('400 with error message', async () => {
-        await assertThrowsAsync(async () => await sendTestRequest({ status: 400, body: 'oops' }), /oops/);
+        await assertThrowsAsync(async () => await sendTestRequest({ statusCode: 400, body: 'oops' }), /oops/);
     });
 
     test('400 with json error message', async () => {
-        await assertThrowsAsync(async () => await sendTestRequest({ status: 400, body: '{ "message": "oops" }' }), (err: Error) => err.message.includes('oops') && !err.message.includes('message'));
+        await assertThrowsAsync(async () => await sendTestRequest({ statusCode: 400, body: '{ "message": "oops" }' }), (err: Error) => err.message.includes('oops') && !err.message.includes('message'));
     });
 
     test('ECONNRESET', async () => {
         await assertThrowsAsync(async () => await sendTestRequest(res => res.destroy()), /socket hang up/i);
     });
-
-    // test('operationSpec with unexpected  error', async () => {
```

Comment:
As I mentioned, we don't have `operationSpec` as a property anymore, so I removed these tests. The errors above basically test the same thing, which was the `StatusCodePolicy`